src/controllers/tripsController.js

```diff
@@ -428,4 +428,24 @@ export default class requestController {
       return Response.errorResponse(res, 500, res.__('server error'));
     }
   }
+
+  /**
+   * @description this function finds and displays most travelled destinations
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} most travelled destinations
+   */
+  static async mostTravelledDestiantions(req, res) {
+    try {
+      const counting = await db.Request.findAll({
+        attributes: ['destination', [db.sequelize.fn('count', db.sequelize.col('destination')), 'count']],
+        group: ['Request.destination'],
+        raw: true,
+        order: db.sequelize.literal('count DESC')
+      });
+      return Response.success(res, 200, res.__('Most travelled destination'), { firstDestiantion: counting[0], secondDestination: counting[1], thirdDestination: counting[2] });
```

Comment:
hey @bbaime98  there is a typo here
```suggestion
      return Response.success(res, 200, res.__('Most travelled destination'), { firstDestination: counting[0], secondDestination: counting[1], thirdDestination: counting[2] });
```

Reply:
it is fixed.